pyproject.toml

```diff
@@ -89,9 +91,9 @@ extra-dependencies = [
   "arrow>=1.3.0",                              # Jinja2TimeExtension
 
   # NamedEntityExtractor
-  "spacy>=3.7,<3.8",
+  "spacy>=3.8,<3.9",
   "spacy-curated-transformers>=0.2,<=0.3",
-  "en-core-web-trf @ https://github.com/explosion/spacy-models/releases/download/en_core_web_trf-3.7.3/en_core_web_trf-3.7.3-py3-none-any.whl",
+  "en-core-web-trf @ https://github.com/explosion/spacy-models/releases/download/en_core_web_trf-3.8.0/en_core_web_trf-3.8.0-py3-none-any.whl",
```

Comment:
The previous `spacy` dependencies required `numpy<2`, while these require `numpy>=2`